klock/src/commonMain/kotlin/com/soywiz/klock/Time.kt

```diff
@@ -449,7 +449,7 @@ inline val Number.minutes get() = TimeDistance(minutes = this.toDouble())
 @Suppress("DataClassPrivateConstructor")
 inline class TimeSpan(private val ms: Double) : Comparable<TimeSpan> {
 	val microseconds: Double get() = this.ms * 1000.0
-	val milliseconds: Int get() = this.ms.toInt()
+	val milliseconds: Long get() = this.ms.toLong()
```

Comment:
I try not to encourage/force to use `Long` in sensitive places in Korlibs because they cause boxing on JS. It will take a few more years before BigInt is available in all browsers and used in Kotlin.

My proposal here is to change millisecodns to `Double` instead (matching the inline type/constructor) and add a `millisecondsLong: Long`.